pseudo_dojo/dojo/works.py

```diff
@@ -55,7 +55,7 @@ def write_dojo_report(self, report, overwrite_data=False):
         old_report[dojo_trial][dojo_ecut] = report
         try:
             self.pseudo.write_dojo_report(old_report)
-        except (OSError, IOError, TypeError):
+        except None: #(OSError, IOError, TypeError):
```

Comment:
Please use

try:

except:

if you want to trap all the possible exceptions.
In python3, one cannot use arbitrary objects wit `except`, only subclasses of `BaseException` are allowed